src/pages/kb/open-source/dev-guide/setup.md

```diff
@@ -28,7 +28,7 @@ You install them with pip:
 pip install -r requirements.txt -r requirements_dev.txt
 ```
 
-(We recommend installing them in a virtualenv. For certain data source types you
+(We recommend installing them in a venv. For certain data source types you
```

Comment:
One shorthand is no better than the other. If we change this reference then it might as well be explicit.

```suggestion
(We recommend installing them in a [virtual environment](https://docs.python.org/3.7/tutorial/venv.html). For certain data source types you
```